src/index.html

```diff
@@ -5,8 +5,238 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="menu__wrapper">
```

Comment:
у тебя здесь нет меню
```suggestion
    <header class="header">
```